.github/dependabot.yml

```diff
@@ -31,6 +31,13 @@ updates:
   commit-message:
     prefix: ":seedling:"
     include: scope
+  ignore:
+  # Ignore controller-runtime major and minor bumps as its upgraded manually.
+  - dependency-name: "sigs.k8s.io/controller-runtime"
+    update-types: ["version-update:semver-major", "version-update:semver-minor"]
+  # Ignore k8s major and minor bumps and its transitives modules
+  - dependency-name: "k8s.io/*"
+    update-types: ["version-update:semver-major", "version-update:semver-minor"]
```

Comment:
Sounds like these will still pull in 'patch' bumps? e.g. 0.32.1 to 0.32.2? If so, that's perfect.

It would be good to have a separate rule to bump k8s and controller-runtime. They could probably both be in the same rule. Even if we never actually used it and always did it manually, it would be good to get the notification that there's a new version.

Reply:
Yes patch versions.